src/components/Modal/jsonModal.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '../Button';
+import './style.scss';
+
+export default class JSONModal extends React.Component {
+  constructor(props) {
+    super(props);
+    const { attendees } = this.props;
+    this.state = {
+      attendees: JSON.parse(attendees).members,
```

Comment:
This is the wrong place to do `JSON.parse`. You should do this data pre-processing in the action or the reducer instead of in the component so that `members` is simply provided to be the data that is expected.